### DISCUSSION TITLE
Using skeleton app and issue with `loginAs`

### DISCUSSION THREAD

**Discussion by antidotecms** [orchestral/testbench-dusk#73] [Q&A]
As per title, I have elected to use a skeleton app for my package testing. However when running tests, I run into the follwing error:

```
Symfony\Component\Routing\Exception\RouteNotFoundException : Route [dusk.login] not defined.
```

My test is:

```
$user = UserFactory::new()->create();

$this->browse(function($browser) use ($user) {
    $browser->loginAs($user, 'web')
        ->visit('/')
        ->assertSee('home');
}
```
Testcase uses:
```
protected function setUp(): void
{
    parent::setUp();

    $this->tweakApplication(function ($app, $config) {
        $app['router']->get('/', fn() => "home");
    });
}

protected function getBasePath()
{
    return __DIR__.'/laravel';
}
```
The skeleton was simply copied from `vendor/orchestra/testbench-dusk/laravel`.

When running the test, the browser starts however it hangs minimized. When it is resized manually, the test completes with the error above. Are there further steps needed to make this work? If I don't use a custom skeleton, the test passes without issue. Any help appreciated!

**Comment by crynobone** [comment]
Seems to me you trying to run the test on `production` environment where Laravel Dusk doesn't add the route for security.

**Reply by antidotecms** [reply]
If I insert `dump(env('APP_ENV'));` at the start of my test, I get `testing` as defined in my phpunit.xml:

```
<env name="APP_ENV" value="testing"/>
```

Really unsure what I might be missing here.
